Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -82,12 +80,72 @@ def tap_new
 
             - run: brew test-bot --only-formulae
               if: github.event_name == 'pull_request'
+
+            - name: Upload bottles as artifact
+              if: always() && github.event_name == 'pull_request'
+              uses: actions/upload-artifact@main
+              with:
+                name: bottles
+                path: '*.bottle.*'
+    YAML
+
+    actions_publish = <<~YAML
+      name: brew pr-pull
+      on:
+        workflow_dispatch:
+          inputs:
+            pull_request:
+              description: Pull request number
+              required: true
+      jobs:
+        pr-pull:
+          runs-on: ubuntu-latest
+          steps:
+            - name: Set up Homebrew
+              uses: Homebrew/actions/setup-homebrew@master
+
+            - name: Set up git
+              uses: Homebrew/actions/git-user-config@master
+
+            - name: Pull bottles
+              env:
+                HOMEBREW_GITHUB_API_TOKEN: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
```

Comment:
Can we use the default `GITHUB_TOKEN` here? Means less setup for tap maintainers.

Reply:
Or at least something clever like using it unless it's a PR from a fork?